src/resources/scss/admin.scss

```diff
@@ -87,6 +87,7 @@
 @import "components/badge";
 @import "components/btn";
 @import "components/card";
+@import "components/checkbox-toggle-button";
```

Comment:
Remove `checkbox-toggle-button` entry and file, move code to end of  `components/_zaa-checkbox-array.scss`.